fsharp-backend/src/LibBackend/TraceInputs.fs

```diff
@@ -27,7 +27,12 @@ module AT = LibExecution.AnalysisTypes
 module RT = LibExecution.RuntimeTypes
 
 
-// space, path, modifier
+/// (space/module, path, modifier)
+///
+/// "Space" is "HTTP", "WORKER", "REPL", etc.
+///
+/// "Modifier" options differ based on space.
+/// e.g. HTTP handler may have "GET" modifier.
```

Comment:
Worth noting that Handlers which don't have modifiers (eg repl, worker) nearly always (but not actually always) have `_` as their modifier.